.github/ISSUE_TEMPLATE/001_bug_report.yml

```diff
@@ -0,0 +1,81 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+type: "Bug"
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+
+      placeholder: |
+        Please also mention any information which could help others to understand
+        the problem you're facing:
+          - What target platform are you using?
+          - What have you tried to diagnose or workaround this issue?
+          - Is this a regression? If yes, have you been able to "git bisect" it to a
+            specific commit?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Steps to reproduce the behavior.
+
+      placeholder: |
+        Steps to reproduce the behavior:
+          1. mkdir build; cd build
+          2. cmake -DBOARD=board\_xyz
+          3. make
+          4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Will it be rendered in the browser as monospace? If so, then I think the second sentence can be removed

Reply:
yeah, the info there is to not have people adding backticks, the content will be rendered automatically.